pkg/component/data/pinecone/v0/component_test.go

```diff
@@ -253,64 +229,4 @@ func TestComponent_Execute(t *testing.T) {
 		})
 	}
 
-	c.Run("nok - 400", func(c *qt.C) {
```

Comment:
These error tests should work with other tasks, too. We might not need to remove them.